pkg/util/kubernetes/hpa/hpa_test.go

```diff
@@ -0,0 +1,157 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build kubeapiserver
+
+package hpa
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"encoding/json"
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/api/core/v1"
+)
+
+func newMockConfigMap(metricName string, labels map[string]string) *v1.ConfigMap {
+	cm := &v1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "ConfigMap",
+		},
```

Comment:
I don't think you need to specify `TypeMeta`, or do you for the fake client?

Reply:
I just checked and that seems to be the case. I did it out of clarity but it might be redundant.